self-api/src/main/java/com/selfxdsd/api/Stars.java

```diff
@@ -0,0 +1,14 @@
+package com.selfxdsd.api;
+
+/**
+ * Repo stars.
+ */
+public interface Stars {
+
+    /**
+     * Star a repository for the authenticated user.
+     * @return True or false, whether the starring
+     *  was successful or not.
+     */
+    boolean star();
```

Comment:
@ak2703 Can you rename this method to ``add()``? Starring a repo should look like this: ``repo.stars().add()``.